packages/models/src/lesson/types.ts

```diff
@@ -6,6 +6,10 @@ import { Step } from '../step';
 import { Activity } from '../activity';
 
 export const TYPE_LESSON = 'lessons';
+export enum TYPE_SUBJECT {
```

Comment:
I think you can remove this one, maybe in `activity` model it makes sense to extend subject type?

Reply:
That was my intention but we'll get to that.